core/plugins/org.polarsys.capella.core.ui.semantic.browser/src/org/polarsys/capella/core/ui/semantic/browser/IImageKeys.java

```diff
@@ -39,4 +39,13 @@
    */
   public final static String IMG_PRIMITIVE_VARIABLES = "variable_tab.gif";
 
+  /**
+   * Image id used in semantic browser to action "lexicographic sort" when sorting is enabled.<br>
+   */
+  public final static String IMG_LEXICOGRAPHIC_SORT_ENABLED = "full/clcl16/capella_sort.gif"; //$NON-NLS-1$
```

Comment:
can you also update the licence.txt file in the project with a reference towards the original core.ui.resource image